specs/windows/shortcut_files.table

```diff
@@ -0,0 +1,27 @@
+table_name("shortcut_files")
```

Comment:
Do we want to prefix windows tables with `windows_` ?

Reply:
i can add a prefix, but the table is only for windows?
and shortcut files seem to be common when using windows
But if u think it will add clarity, i can add it